src/redux/sagas/user-search.saga.js

```diff
@@ -0,0 +1,19 @@
+import axios from 'axios';
+import { put, takeLatest } from 'redux-saga/effects';
+
+function* fetchCompanies(action) {
+    try {
+        const searchCompanies = yield axios.get(`/api/location/${action.payload}`);
```

Comment:
I think this has to be passed as a query. each item will have to be pulled out of action.payload and put as a query value so it'll look like:
/api/location/?state=${action.payload.state}&metaldrums={$action.payload.metaldrums}& .....